src/PageLayout/PageLayout.tsx

```diff
@@ -370,6 +433,8 @@ export type PageLayoutPaneProps = {
    */
   positionWhenNarrow?: 'inherit' | keyof typeof panePositions
   width?: keyof typeof paneWidths
+  canResizePane?: boolean
```

Comment:
What do you think about calling this prop `resizable`?

I think `<PageLayout.Pane resizable>` reads well.

Reply:
Yes, good point. Done in https://github.com/primer/react/pull/2352/commits/56413f8de76ef2b50c20347869e2c5beb202daad